aptos-move/framework/src/natives/code.rs

```diff
@@ -280,7 +280,7 @@ fn native_request_publish(
         bundle: ModuleBundle::new(code),
         expected_modules,
         allowed_deps,
-        check_compat: policy == CHECK_COMPAT_POLICY,
+        check_compat: policy != ARBITRARY_POLICY,
```

Comment:
Let's leave a comment that this is actually not used: https://github.com/aptos-labs/aptos-core/blob/163b335b841510a2e9afc259dd8088b0d58f8746/aptos-move/aptos-vm/src/aptos_vm.rs#L565. But we still do roughly the right thing here for readability?

Reply:
I dunno, I feel like these type of comments are not ever updated...

if we really want a comment it belongs on line 565, but there it is rather obvious...

Reply:
Why don't we just kill this whole check_compat variable altogether?

Reply:
This should be `policy != UpgradePolicy::arbitrary()`. This constant should not have been there in the first place.

Reply:
> Why don't we just kill this whole check_compat variable altogether?

Can you please in the future just open an issue for something like this and assign to the code owner? It is Sunday, and this doesn't deserve the trouble made around it.